### ISSUE TITLE
ng build fails when less compiler deals with inline javascript

### ISSUE THREAD

**Issue by cafesanu** [ng-packagr/ng-packagr#1316]
## Type of Issue

```
[ X] Bug Report
[ ] Feature Request
```

## Description
  - Similar to why this issue was opened (https://github.com/angular/angular-cli/issues/10430) and what this commit solved (https://github.com/angular/angular-cli/pull/11958), Angular libraries compiled via ng-packagr are still having this issue.
  - I think the issue would be solved by adding the `--js` flag to this line https://github.com/ng-packagr/ng-packagr/blob/master/src/lib/ng-v5/entry-point/resources/stylesheet-processor.ts#L74

_A bug: please describe the error that you encountered_
```
Error: Command failed: node "/Users/carlos/dev/design-system/node_modules/less/bin/lessc" "/Users/carlos/dev/design-system/projects/design-system/src/lib/components/drawer/drawer.component.less" --less-plugin-npm-import="prefix=~"
SyntaxError: Inline JavaScript is not enabled. Is it set in your options? in /Users/carlos/dev/design-system/node_modules/ng-zorro-antd/style/color/bezierEasing.less on line 110, column 1:
109 // https://github.com/ant-design/ant-motion/issues/44
110 .bezierEasingMixin();
111


    at checkExecSyncError (child_process.js:616:11)
    at Object.execSync (child_process.js:653:13)
    at StylesheetProcessor.renderPreProcessor (/Users/carlos/dev/design-system/node_modules/ng-packagr/lib/ng-v5/entry-point/resources/stylesheet-processor.js:71:40)
    at StylesheetProcessor.process (/Users/carlos/dev/design-system/node_modules/ng-packagr/lib/ng-v5/entry-point/resources/stylesheet-processor.js:40:34)
    at Object.readResource (/Users/carlos/dev/design-system/node_modules/ng-packagr/lib/ts/cache-compiler-host.js:73:57)
    at TsCompilerAotCompilerTypeCheckHostAdapter.loadResource (/Users/carlos/dev/design-system/node_modules/@angular/compiler-cli/src/transformers/compiler_host.js:495:37)
    at Object.get (/Users/carlos/dev/design-system/node_modules/@angular/compiler/bundles/compiler.umd.js:26425:94)
    at DirectiveNormalizer._fetch (/Users/carlos/dev/design-system/node_modules/@angular/compiler/bundles/compiler.umd.js:18265:47)
    at /Users/carlos/dev/design-system/node_modules/@angular/compiler/bundles/compiler.umd.js:18387:72
    at Array.map (<anonymous>)
    at DirectiveNormalizer._loadMissingExternalStylesheets (/Users/carlos/dev/design-system/node_modules/@angular/compiler/bundles/compiler.umd.js:18387:18)
    at DirectiveNormalizer._normalizeTemplateMetadata (/Users/carlos/dev/design-system/node_modules/@angular/compiler/bundles/compiler.umd.js:18335:40)
    at /Users/carlos/dev/design-system/node_modules/@angular/compiler/bundles/compiler.umd.js:18292:116
    at Object.then (/Users/carlos/dev/design-system/node_modules/@angular/compiler/bundles/compiler.umd.js:2470:81)
    at DirectiveNormalizer.normalizeTemplate (/Users/carlos/dev/design-system/node_modules/@angular/compiler/bundles/compiler.umd.js:18292:30)
    at CompileMetadataResolver.loadDirectiveMetadata (/Users/carlos/dev/design-system/node_modules/@angular/compiler/bundles/compiler.umd.js:20541:62)

```
_A feature: please describe your use case and motivation_
This is happening because I'm using the wildly used ng-antd-zorro library, which happens to use inline javascript.

#### How To Reproduce

_A bug: please include instructions how to reproduce. Issues without reproduction are likely to receive no feedback._

In your component, try to include any less file that had inline javascript (like `node_modules/ng-zorro-antd/ng-zorro-antd.less`)

_Can you reproduce the error in the integration tests in ng-packagr?_
_If possible, take a look at the [`integration/samples`](./integration/samples) and try to break one of these builds!_

_Is the error you faced in an application importing the library_
_Try to break the Angular CLI app in [`integration/consumers/ng-cli`](./integration/consumers/ng-cli)!_

#### Expected Behaviour

_A bug: please describe what behaviour or result you expected_
`ng build` should not fail
_A feature: do you have a first draft or an idea how to implement?_

#### Version Information

```
$ node_modules/.bin/ng-packagr --version:
ng-packagr: 5.2.0
@angular/*: 8.0.0
rollup: 1.13.1
tsickle: 0.35.0
typescript: 3.4.5
```

_Please include all version numbers that might be relevant, e.g. third-party libraries_

**Comment by yy7054wyq5** [comment]
edit this file node_modules/ng-packagr/lib/ng-v5/entry-point/resources/stylesheet-processor.js
```javascript
let cmd = `node node_modules/less/bin/lessc "${filePath}" --less-plugin-npm-import="prefix=~" --js`;
```

**Comment by alan-agius4** [comment]
Solved in 5.3.0
